src/frontend/screens/SyncModal/index.js

```diff
@@ -41,10 +42,17 @@ class SyncModal extends React.Component<Props, State> {
   state = { serverPeers: [], syncErrors: new Map(), ssid: null };
   _opened: number;
   _subscriptions: Array<{ remove: () => void }> = [];
+  _deviceName: string;
+
+  constructor (props) {
+    super(props)
+    // recreate this as little as possible
+    this._deviceName = this._deviceName || "Android " + crypto.randomBytes(2).toString('hex').slice(0, 3);
```

Comment:
Ideally the device name wouldn't change every time you open/close sync

Reply:
I switched to generate this once during the lifecycle of the app
FYI whenever the constructor is called, this._deviceName would always be undefined, since it's only called when creating a new instance.